### PR TITLE
Alemata.react customizing lock

### PR DESCRIPTION
**Pull request by alemata**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by alemata** [comment]
@flaviogranero could you review please? Sorry 👎

**Comment by flaviogranero** [comment]
👍 @chenkie @mpaktiti

File: articles/client-platforms/react/10-customizing-lock.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+title: Customizing Lock
+description: This tutorial will show you how to customize lock widget.
+---
+
```
Comment (mpaktiti) [inline articles/client-platforms/react/10-customizing-lock.md:5]: @alemata Missing system requirements:

```
::: panel-info System Requirements
This tutorial and seed project have been tested with the following:
* NodeJS 5.2.0
* NPM 3.3.12
* React 15.0.2
:::
```

Comment (alemata) [inline articles/client-platforms/react/10-customizing-lock.md:5]: updated!

**Pull request merged**